activestorage/test/service/configurations.yml

```diff
@@ -0,0 +1,30 @@
+s3:
+  service: S3
+  access_key_id: <%= ENV["AWS_ACCESS_KEY_ID"] %>
+  secret_access_key: <%= ENV["AWS_SECRET_KEY"] %>
+  region: us-east-2
+  bucket: rails-ci-activestorage
```

Comment:
Because of these lines, tests are run by default against S3 but not against GCS and Azure.

However, a rails developer might not have access to an S3 instance to make these tests pass.

How do you feel about having something on the line of Active Record's tests, where you can run:

```
bundle exec rake test:s3
bundle exec rake test:gcs
bundle exec rake test:azure
```

if you explicitly want to test those components, whereas as default you only test disk storage?